data-client/build.gradle

```diff
@@ -55,6 +62,7 @@ android {
         sourceCompatibility = '1.8'
         targetCompatibility = '1.8'
     }
+    namespace 'org.wikipedia.dataclient'
```

Comment:
Added the namespace DSL to the android {} block in the module-level build.gradle file and removed the package attribute in the manifest.

This change was made as part of preparing the app build for the Android Studio Flamingo release.

Refer to this [medium article](https://medium.com/androiddevelopers/5-ways-to-prepare-your-app-build-for-android-studio-flamingo-release-da34616bb946).